doorrando/normal.asm

```diff
@@ -416,4 +416,28 @@ InroomStairsTrapDoor:
     pla : pla : pla
     jsl StraightStairsTrapDoor_reset
     jml $028b15 ; just some RTS in bank 02
+}
+
+HandleSpecialDoorLanding: {
+    LDA.l $7F2000,X ; what we wrote over
+    SEP #$30
+    JSL HandleIncomingDoorState
+    CMP #$34 : bne + ; inroom stairs
+        PHA : LDA #$26 : STA $045E : PLA
+    +
+}
```

Comment:
Missing RTL?

Reply:
Looks like it. I was wondering why it still works as is and doesn't crash as it is. And then I realized that it's essentially calling HandleIncomingDoorState twice and it's RTLing there. I've amended the commit to include this.